stix/info.json

```diff
@@ -54,13 +54,77 @@
       ],
       "parameters": [
         {
-          "title": "Stix2 File IRI",
+          "title": "Stix2 FileIRI or Filename",
           "required": true,
           "editable": true,
           "visible": true,
           "type": "text",
           "name": "file_id",
-          "description": "Specify Stix2 Json File IRI"
+          "tooltip": "Specify Stix2 Json FileIRI or FileName under /tmp folder",
```

Comment:
File containing the input in STIX2 format. You may provide the IRI if the file is uploaded to FortiSOAR, or the name of the file uploaded under /tmp on the server